docs/content.zh/docs/deployment/overview.md

```diff
@@ -152,7 +152,14 @@ When deploying Flink, there are often multiple options available for each buildi
     </tbody>
 </table>
 
-
+### Repeatable Resource Cleanup Strategy
+
+Once a job has reached a globally terminal state of either finished, failed or cancelled, the
+external component resources associated with the job are then cleaned up. In the event of a
+failure when cleaning up a resource, Flink will attempt to retry the cleanup based on
+a repeatable retry strategy. You can [configure]({{< ref "docs/deployment/config" >}}) this
```

Comment:
Ah, I think I tried to assign a link to an anchor before, but it didnt work - but I think its because i used `docs/deployment/config#anchor` rather than `config/#anchor`. I'll change this and see if it works correctl|y

Reply:
Yep, it works - the syntax is `/docs/deployment/config#retryable-cleanup`.